packages/Calendar/src/DayView/Day.js

```diff
@@ -55,7 +49,6 @@ function Day({
   const dateObj = new Date(date);
   const isCurrentMonth = currentMonth === dateObj.getMonth();
   const isToday = dateObj.toDateString() === new Date().toDateString();
-
   return (
     <TouchableOpacity
       style={[styles.container, borderStyle || { borderColor: 'transparent' }]}
```

Comment:
The transparent borderColor could be part of styles.container

Reply:
that creates an issue in android